### ISSUE TITLE
Instructions for Starting Server at least on MacOS

### ISSUE THREAD

**Issue by dgentry** [MisterTea/EternalTerminal#457]
It was a little hard to figure out what I was supposed to do to start a server on MacOS.

On Ubuntu, installing the package from the ppa starts a server.

On MacOS Big Sur (m1), I don't seem to get a running server after the `brew install`.
On MacOS Mojave (x86), building `et 6.0.13` from source doesn't seem to get me a server either.
On Debian, building from source doesn't start a server, but I figured out that I could run `etserver` from the command line (because I saw `etserver` as one of the build products).

On Debian, I think we just need to add a line to the instructions in the README:

```
sudo sed 's:/usr/bin:/usr/local/bin/:' ../systemctl/et.service >/etc/systemd/system/et.service
```

On MacOS, I think the instructions would be something like:

```
sudo cp ../init/launchd/homebrew.mxcl.et.plist /Library/LaunchDaemons/
sudo launchctl load /Library/LaunchDaemons/homebrew.mxcl.et.plist
sudo launchctl start /Library/LaunchDaemons/homebrew.mxcl.et.plist
sudo launchctl list | grep homebrew.mxcl.et
```

For m1 Macs, replace the cp with: 

```
sudo sed 's:/usr/local/bin/etserver:/opt/homebrew/bin/etserver' ../init/launchd/homebrew.mxcl.et.plist >/Library/LaunchDaemons/homebrew.mxcl.et.plist
```

If you think this looks reasonable, I'll make a pull request adding this stuff to the README.

**Comment by MisterTea** [comment]
I'm changing the default installation directory on Linux to /usr/bin in the next version, but the Mac instructions are great. Please submit a PR!

**Comment by dgentry** [comment]
#460 fixes this (and doesn't say anything about Debian).  Please review and commit if you like it.

**Comment by dgentry** [comment]
Fixed by #460.
